packages/contract-wrappers/src/contract_wrappers/dutch_auction_wrapper.ts

```diff
@@ -14,12 +14,11 @@ import { DutchAuctionWrapperError, OrderTransactionOpts } from '../types';
 import { assert } from '../utils/assert';
 import { _getDefaultContractAddresses } from '../utils/contract_addresses';
 
-import { ContractWrapper } from './contract_wrapper';
-
-export class DutchAuctionWrapper extends ContractWrapper {
+export class DutchAuctionWrapper {
     public abi: ContractAbi = DutchAuction.compilerOutput.abi;
     public address: string;
-    private _dutchAuctionContractIfExists?: DutchAuctionContract;
+    protected _web3Wrapper: Web3Wrapper;
```

Comment:
Private. Protected means if a class inherits from `DutchAuctionWrapper`, it will be able to access `_webWrapper` which doesn't seem like what we want.